module_11/data/authtorization.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$authtirized = false;
+
+if (array_key_exists('login', $_POST)) {
```

Comment:
`array_key_exists` is a very expensive operation. If you will make array of user up side down, you will get an array like
`
[
'login1' => 1,
'login2' => 2
]
`
In this case you can easy compare login and find the key in the array
`$key = $revertedArray[$_POST['login']] ?? null;`
after that you can try to find password and compare it
`$authtorized = $password[$key] === $_POST['password']`

Reply:
in this case i should to use array_flip()?

Reply:
okey, i will use empty